src/Menu/MenuBuilderInterface.php

```diff
@@ -19,7 +19,7 @@
 /**
  * @author Christian Gripp <mail@core23.de>
  *
- * @deprecated since 3.x, to be removed with 4.0.
+ * @deprecated since 3.9, to be removed with 4.0.
```

Comment:
I don't know why this changed, but It doesn't matter I guess, since this will be removed in the end.